<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="QT-INBG-LT0263" failures="1" tests="1" name="Test" time="21.251" errors="0" timestamp="2022-10-17T15:15:41 IST">
  <testcase classname="ppom.Defect" name="verifyLenovoOrderPage" time="21.076">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [https://qualicoach.org/mwapp/mobile.html]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [https://qualicoach.org/mwapp/mobile.html]
at ppom.Defect.verifyLenovoOrderPage(Defect.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyLenovoOrderPage -->
  <testcase classname="ppom.Defect" name="verifyLenovoTableTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLenovoTableTest -->
  <testcase classname="ppom.Defect" name="defect1" time="0.0">
    <ignored/>
  </testcase> <!-- defect1 -->
  <testcase classname="ppom.Defect" name="defect" time="0.0">
    <ignored/>
  </testcase> <!-- defect -->
</testsuite> <!-- Test -->
